### ISSUE TITLE
Slowfast: Output is uniformly zero; network calibration failed.

### ISSUE THREAD

**Issue by kunalgoyal9** [NVIDIA/TensorRT#663]
I was trying to do INT8 quantization for the SlowFast model. I have successfully generated the calibration file and seems ok to me. But while running that I got failure for some nodes in my graph.

like this
 
`
[TensorRT] WARNING: Tensor (Unnamed Layer* 371) [Constant]_output is uniformly zero.
[TensorRT] WARNING: Tensor (Unnamed Layer* 372) [Shuffle]_output is uniformly zero.
[TensorRT] WARNING: Tensor 1003 is uniformly zero.
[TensorRT] WARNING: Tensor 1013 is uniformly zero.
[TensorRT] WARNING: Tensor 1023 is uniformly zero.
[TensorRT] WARNING: Tensor 685 is uniformly zero.
[TensorRT] WARNING: Tensor 695 is uniformly zero.
[TensorRT] WARNING: Tensor 705 is uniformly zero.
[TensorRT] WARNING: Tensor 717 is uniformly zero.
[TensorRT] WARNING: Tensor 727 is uniformly zero.
[TensorRT] WARNING: Tensor 737 is uniformly zero.
[TensorRT] WARNING: Tensor 755 is uniformly zero.
[TensorRT] WARNING: Tensor 765 is uniformly zero.
[TensorRT] WARNING: Tensor 775 is uniformly zero.
[TensorRT] WARNING: Tensor 785 is uniformly zero.
[TensorRT] WARNING: Tensor 797 is uniformly zero.
[TensorRT] WARNING: Tensor 807 is uniformly zero.
[TensorRT] WARNING: Tensor 817 is uniformly zero.
[TensorRT] WARNING: Tensor 827 is uniformly zero.
[TensorRT] WARNING: Tensor 843 is uniformly zero.
[TensorRT] WARNING: Tensor 853 is uniformly zero.
[TensorRT] WARNING: Tensor 863 is uniformly zero.
[TensorRT] WARNING: Tensor 873 is uniformly zero.
[TensorRT] WARNING: Tensor 883 is uniformly zero.
[TensorRT] WARNING: Tensor 893 is uniformly zero.
[TensorRT] WARNING: Tensor 905 is uniformly zero.
[TensorRT] WARNING: Tensor 915 is uniformly zero.
[TensorRT] WARNING: Tensor 925 is uniformly zero.
[TensorRT] WARNING: Tensor 935 is uniformly zero.
[TensorRT] WARNING: Tensor 945 is uniformly zero.
[TensorRT] WARNING: Tensor 955 is uniformly zero.
[TensorRT] WARNING: Tensor 971 is uniformly zero.
[TensorRT] WARNING: Tensor 981 is uniformly zero.
[TensorRT] WARNING: Tensor 991 is uniformly zero
`
 
When I tried to visualize those tensors in Netron. They belong to the output of batch norm layer above *ADD* node

![image](https://user-images.githubusercontent.com/21281457/86512593-185a7100-be21-11ea-9a44-ccdca065a276.png)

**Questions:**

I didn't fully understand the warning. If I am getting some tensor uniformly zero what is wrong while calibration? 

## Environment

**TensorRT Version: 7.1**: 
**GPU Type: Tesla T4**:  
**CUDA Version: 10.2**:  
**Operating System: Ubuntu 18.04**: 
**Python Version (if applicable): python3**:  
**PyTorch Version (if applicable): 1.5**: 
**Container: https://ngc.nvidia.com/catalog/containers/nvidia:tensorrt**:

## Relevant Files
Calibration file: [link](https://gist.github.com/kunalgoyal9/1cc1079693786ec633496b8d3e810bd6)
Running Int8 architecture file: [link](https://gist.github.com/kunalgoyal9/48701e8275766cb3c4f13538ce11dfd4)

Model link:

## Steps To Reproduce

1.) Convert SlowFast model to onnx format.
2.) Use Calibration File mention above to calibrate on UCF101 data
3.) Run model using file mentioned above.

**Comment by ttyio** [comment]
Hello @kunalgoyal9 , TensorRT would check all the intermediate layer outputs during calibration stage, and the `uniformly zero` warning means that we find these tensors output are zeros during calibration given the calibration data you feed. 
Could you try feeding with a different dataset on calibration? or try to mark these layer as network output, and see if they are really zeros in an FP32 run.  thanks.

**Comment by ttyio** [comment]
I will close this issue since no response for more than 3 weeks, please reopen if you still have question, thanks!
